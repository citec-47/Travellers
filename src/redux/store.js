import { configureStore } from '@reduxjs/toolkit';
import rocketsReducer from './rocket/rocketSlice';
import missionReducer from './mission/misionSlice';

const store = configureStore({
  reducer: {
    allRockets: rocketsReducer,
    mission: missionReducer,
  },
});
export default store;
