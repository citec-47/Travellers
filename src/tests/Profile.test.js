import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import Myprofile from '../components/Profile';
import store from '../redux/store';

describe('Test Profile components', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Myprofile />
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
