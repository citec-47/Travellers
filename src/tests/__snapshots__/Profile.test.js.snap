// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Profile components renders correctly 1`] = `
<div
  className="mycontainer"
>
  <div
    className="ourMission"
  >
    <h2>
      My Missions
    </h2>
    <ul />
  </div>
  <table />
</div>
`;
